packages/lesswrong/splits/react-map-gl.ts

```diff
@@ -0,0 +1,3 @@
+import ReactMapGL, { Marker, Popup } from 'react-map-gl';
+
+(window as any).reactMapGL = { ReactMapGL, Marker, Popup };
```

Comment:
Could you use a context manager here rather than putting it on the window. If we have more than a couple of these I wouldn't want them clogging up the global namespace

Reply:
The issue is that this is running in from a different script in an entirely different compilation unit, which needs to pass data to the main bundle somehow. Once it's inside the main bundle, it could use React context, but I don't think there's any way to do the first step that way.